cli/item_to_function.py

```diff
@@ -117,6 +117,18 @@ def _get_item_yaml(item_path: Path) -> dict:
     item_yaml = full_load(open(item_path, "r"))
     return item_path, item_yaml
 
+def _get_function_yaml(function_path: Path) -> dict:
+    if function_path.is_dir():
+        if (function_path / "function.yaml").exists():
+            function_path = function_path / "function.yaml"
+        else:
+            raise FileNotFoundError(f"{function_path} does not contain a item.yaml file")
+    elif not function_path.exists():
+        raise FileNotFoundError(f"{function_path} not found")
+
+    item_yaml = full_load(open(function_path, "r"))
+    return function_path, item_yaml
+
```

Comment:
Looks like this is not used, why is this here?

Reply:
will remove it